miner/worker.go

```diff
@@ -593,6 +597,7 @@ func (w *worker) resultLoop() {
 				log.Error("Failed writing block to chain", "err", err)
 				continue
 			}
+			blockFinalizationTimeGauge.Update(time.Now().UnixNano() - int64(block.Time())*1000000000)
```

Comment:
How come this is using nanoseconds scale?

Reply:
Two factors:
• block.Time() will return seconds.
• time.Now() will return seconds or nanoseconds.

For accuracy, I chose to use nanoseconds.

Reply:
:+1: